bin/php-scoper

```diff
@@ -10,17 +10,20 @@
  * file that was distributed with this source code.
  */
 
-use Webmozart\PhpScoper\Console\Application;
-use Webmozart\PhpScoper\Console\ApplicationConfig;
+use PackageVersions\Versions;
+use Symfony\Component\Console\Application;
+use Humbug\PhpScoper\Console\Command\AddPrefixCommand;
 
 if (file_exists($autoload = __DIR__.'/../../../autoload.php')) {
     require_once $autoload;
 } else {
     require_once __DIR__.'/../vendor/autoload.php';
 }
 
-$app = new Application();
-$conf = new ApplicationConfig();
-$conf->configure($app);
+$app = new Application('php-scoper', Versions::getVersion('humbug/php-scoper'));
```

Comment:
Add test somewhere for output? Specifically, down the line for phar builds, this should be returning a SemVer compatible string, and the current Ocramius package does not (e.g. uses `@` to append metadata instead of `+`). When building Phars, Box has a similarly incompatible approach by not appending `-dev` for builds which are a tagged version + additional commits.